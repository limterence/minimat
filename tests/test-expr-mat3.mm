/* test-expr-mat3: matrix assignment, subselect, definition */
int main() {
  matrix a;
  a = [1.0; 2.0;];   /* 2 x 1 matrix */
  a = [a; a';];      /* cannot vertcat matrices with different columns */
}
