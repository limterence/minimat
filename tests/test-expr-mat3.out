Matrices cannot have different col size.  Exiting...
